base-genesis.json

```diff
@@ -108,7 +108,10 @@
         "max_deposit_period": "1209600s"
       },
       "voting_params": {
-        "voting_period": "31536000s"
+        "voting_period_default": "172800s",
```

Comment:
I would make it 14 days minimum tbh.
2 seems low

Reply:
Yeah I agree, I forgot to think about the value, which is the one defined for the default period the legacy module. Will have to update https://github.com/atomone-hub/govgen/pull/16 first.

Reply:
oh silly me, we can just replace it there.

Reply:
done [1140a7b](https://github.com/atomone-hub/govgen-genesis/pull/1/commits/1140a7b633497b2cdee9647f05ecc3a70e271eb7)